### PR TITLE
Visitor Generator plugin language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
### Description:

Please include a description of this change and which issue it fixes. If no issue exists yet please include context and what problem it solves.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
         "GenerateFakeActions": "This will generate at least %d fake actions on this site for each day.",
-        "OverwriteLogFiles": "You can overwrite the log files that are used to generate fake visits (change or add files in %s). There are log files of requests to \"matomo.php\" in the format \"Apache combined log\". Plugin developers can provide their own log files by placing '*.log' files within a 'data' directory of their plugin.",
-        "PleaseBePatient": "NOTE: It might take a few minutes to generate visits and actions, please be patient...",
-        "LogImporterNote": "There is also a faster tool that will import large test data in Matomo, see the %1$sREADME%2$s.",
-        "AutomaticReprocess": "The reports will be reprocessed the next time you visit the Matomo reports, it might take a few minutes.",
+        "OverwriteLogFiles": "You can overwrite the log files used to generate fake visits (change or add files in %s). There are log files of requests to \"matomo.php\" in the format \"Apache combined log\". Plugin developers can provide their own log files by placing '*.log' files within a 'data' directory of their plugin.",
```
Comment (comradekingu) [inline lang/en.json]:
```suggestion
        "OverwriteLogFiles": "Overwrite the \"matomo.php\" log files to generate fake visits in the \"Apache combined log\" format (change or add files in %s). Plugin developers add their own log files as '*.log' files in a 'data' folder of their plugin.",
```
?

File: lang/en.json [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+        "PleaseBePatient": "Generating visits and actions…",
```
Comment (justinvelluppillai) [inline lang/en.json]:
```suggestion
        "PleaseBePatient": "Generating visits and actions, this may take a few minutes…",
```

**Review by justinvelluppillai — APPROVED** [review]
Looks good. Made one optional suggestion.

File: lang/en.json (hunk 2 above)
Comment (sgiehl) [reply lang/en.json]: That might need a rewording due to the way this is currently used. It is not shown while visits are generated, but as a note, before that is done.

This is how the page using most of the translations here currently looks like:
image

@comradekingu maybe you can think of some further adjustments based on that.

File: lang/en.json [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
         "DaysToCompute": "Days to compute",
         "GenerateFakeActions": "This will generate at least %d fake actions on this site for each day.",
-        "OverwriteLogFiles": "You can overwrite the log files that are used to generate fake visits (change or add files in %s). There are log files of requests to \"matomo.php\" in the format \"Apache combined log\". Plugin developers can provide their own log files by placing '*.log' files within a 'data' directory of their plugin.",
-        "PleaseBePatient": "NOTE: It might take a few minutes to generate visits and actions, please be patient...",
-        "LogImporterNote": "There is also a faster tool that will import large test data in Matomo, see the %1$sREADME%2$s.",
-        "AutomaticReprocess": "The reports will be reprocessed the next time you visit the Matomo reports, it might take a few minutes.",
+        "OverwriteLogFiles": "Overwrite the \"matomo.php\" log files to generate fake visits in the \"Apache combined log\" format (change or add files in %s). Plugin developers add their own log files as '*.log' files in a 'data' folder of their plugin.",
+        "PleaseBePatient": "Generating visits and actions…",
```
Comment (comradekingu) [inline lang/en.json]:
```suggestion
        "PleaseBePatient": "Generating visits and actions can take a few minutes.",
```

**Pull request merged**